packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -611,3 +620,106 @@ function last(walker: TreeWalker) {
   } while (last);
   return next;
 }
+
+
+class Tree {
+  private root: Node;
+  private fastMap = new Map<ScopeRef, Node>();
+
+  constructor() {
+    this.root = new Node({scopeRef: null});
+    this.fastMap.set(null, this.root);
+  }
+
+  get size() {
```

Comment:
i only use this for testing purposes to make to make sure the tree is cleaned up, thoughts?